### PR TITLE
Adding Sprites

### PR DESCRIPTION
**Pull request by mwyoung**
The main things that were updated with this change would be that it is now possible to get sprites (with a method in PokeAPIUtils to create the URL), and a Glide call in PokemonListAdapter. The pokemon_list_entry was also edited to make it look better with larger icons. I also added the ability to get the count, along with a (very simple) formula to get the last index for that Pokemon.

### CODE REVIEW

File: app/src/main/java/rec/games/pokemon/teambuilder/PokemonListAdapter.java [hunk 1]
```diff
@@ -78,6 +81,8 @@ public void bind(Pokemon p)
 	{
 		mName.setText(p.identifier);
 		mId.setText(String.valueOf(p.id));
+		Glide.with(mIcon.getContext()).load(PokeAPIUtils.getSpriteUrl(p.id))
+			.apply(new RequestOptions().placeholder(R.drawable.ic_poke_unknown)).into(mIcon);
```
Comment (magical) [inline app/src/main/java/rec/games/pokemon/teambuilder/PokemonListAdapter.java]: Nice.

**Review by magical — APPROVED** [review]
LGTM. We might want to switch to displaying icons instead of sprites later, but this is good for now.

**Comment by mwyoung** [comment]
I updated Glide to not show errors when it doesn't find an image, along with a few request options. I also removed the conflict.

File: app/src/main/java/rec/games/pokemon/teambuilder/MainActivity.java [hunk 2]
```diff
@@ -29,6 +32,13 @@ protected void onCreate(Bundle savedInstanceState)
 		super.onCreate(savedInstanceState);
 		setContentView(R.layout.activity_main);
 
+		RequestOptions requestOptions = new RequestOptions()
+			.placeholder(R.drawable.ic_poke_unknown)
+			.placeholder(R.drawable.ic_poke_unknown)
+			.placeholder(R.drawable.ic_poke_unknown)
```
Comment (magical) [inline app/src/main/java/rec/games/pokemon/teambuilder/MainActivity.java]: Why three placeholders?

Comment (mwyoung) [reply app/src/main/java/rec/games/pokemon/teambuilder/MainActivity.java]: Those are typos. I am going to fix that right now.

Comment (mwyoung) [reply app/src/main/java/rec/games/pokemon/teambuilder/MainActivity.java]: I updated this with the actual code that I meant. 
https://github.com/OSU-CS492-W19/final-project-team-pikachu/blob/5a924acda119dabc64ce70ef9d3d5bd01f97bec9/app/src/main/java/rec/games/pokemon/teambuilder/MainActivity.java#L36-L38

**Comment by mwyoung** [comment]
@hammockt (or @magical), if you don't have any complaints about this code I was going to merge both pull requests tomorrow. If you do let me know what I need to fix.

**Comment by mwyoung** [comment]
I just fixed a merge conflict as well (from the error message code), but it was with non-conflicting code.

**Pull request merged**